etc/frontend_development.ini.in

```diff
@@ -28,6 +28,14 @@ adhocracy.frontend.template_path = /static/templates
 adhocracy.frontend.rest_url = http://localhost:6541
 # backend behind varnish (default port)
 #adhocracy.frontend.rest_url = http://localhost:8088
+# Only accept registration requests with valid captcha solutions
+adhocracy.captcha.required = ${thentos_captcha:required}
+# Where the frontend sends captcha traffic
+adhocracy.captcha.backend_url = ${thentos_captcha:backend_url}
+# Where the backend sends captcha traffic (in development mode, this
+# is the same as captcha backend; if nginx is dispatching inbound
+# requests, this is just the frontend url)
+adhocracy.captcha.frontend_url = ${thentos_captcha:frontend_url}
 # The email address show to users seeking support
 adhocracy.frontend.support_email = support@unconfigured.domain
 # Default frontend locale
```

Comment:
i think this is a misunderstanding; see one my other comments from a moment ago for an explanation.  `adhocracy.frontend_url` points to the server that delivers the a3 frontend; the above two urls are for a3 frontend and backend to contact the captcha service, respectively.

i think the problem with this PR is that we don't really use the new config values anywhere.  i just thought it would be nice to have these values already present from the config when we start changing the a3 frontend / backend.